next.config.js

```diff
@@ -10,4 +10,16 @@ module.exports = {
     // Will be used for non-localized paths
     defaultLocale: "fi",
   },
+  async rewrites() {
+    return [
+      {
+        source: "/haku",
+        destination: "/search",
+      },
+      {
+        source: "/sok",
+        destination: "/search",
+      },
```

Comment:
If I understand this correctly this is swedish translation for search? If so shouldn't it be `sök`?

Reply:
Cleared this up on Slack 👍